ref/network-ports.md

```diff
@@ -0,0 +1,35 @@
+The following tables list the network ports that Anbox Cloud exposes.
+
+For the regular Anbox Cloud deployment, these port are used for communication between components, and to allow accessing the Anbox Cloud interface. For the Anbox Cloud appliance, all components are installed on the same machine. Therefore, there is no need to expose ports for internal communication.
+
+<a name="anbox-cloud"></a>
+## Anbox Cloud
+
+| Service               | Port(s)     | Protocol | Exposed externally | Required                     | Description               |
+|-----------------------|-------------|----------|--------------------|------------------------------|---------------------------|
+| AMS                   | 8444        | TCP      | no                 | yes                          | HTTPS API                 |
```

Comment:
I changed this (and all other "HTTP") to "HTTPS" - is that okay or is it misleading in any way?

Reply:
Makes sense